client/src/routes/+page.svelte

```diff
@@ -44,9 +44,9 @@
       // Create User
       await saveOrCreate(user, userData, name.trim());
       // Create Lobby
-      const code = await createLobby(name.trim());
+      const responce = await createLobby();
```

Comment:
why?

Reply:
why what?

Reply:
the name is no longer an argument because it is read from the userData doc and the return type is a response object that contains a code rather than being the just a string

Reply:
oh wait

Reply:
you mean the spelling